rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -2512,9 +2512,9 @@ bool opcode_00e3(const ScriptArguments& args, const ScriptPlayer player, const S
 	@arg arg6 
 */
 bool opcode_00e4(const ScriptArguments& args, const ScriptPlayer player, const ScriptVec2 coord0, const ScriptVec2 coord1, const ScriptInt arg6) {
-	auto ply = player->getCharacter();
-	auto condition = ply->getCurrentVehicle() == nullptr;
-	return script::objectInRadius(args, ply, coord0, coord1, arg6, condition);
+	auto plyChar = player->getCharacter();
```

Comment:
`playerChar` or even `playerCharacter` would have been better imo, but this'll do ;)
(I'm not a fan of abbreviations)